packages/block-library/src/post-author/editor.scss

```diff
@@ -1,12 +1,12 @@
-.block-editor__container .wp-block-post-author {
+.block-editor__container .wp-block-post-author,
+.edit-site-visual-editor .wp-block-post-author {
```

Comment:
Why is this using `block-editor__container`. I don't think that selector is used anywhere.

Reply:
Yeah, `.block-editor__container .wp-block-post-author` was the only selector that was there previously.  Its part of the post editor, but I don't know what it was the class originally put here.  To be safe I just added the `.edit-site-visual-editor .wp-block-post-author` selector as well so the styles would be in the site editor as well.

Reply:
I don't even see it anywhere in the DOM. We should just use `.wp-block-post-author`.

Reply:
> I don't even see it anywhere in the DOM. 

Odd, for me its a child of the .block-editor:
![Screen Shot 2020-06-05 at 6 47 22 PM](https://user-images.githubusercontent.com/28742426/83928264-1a1b1100-a75d-11ea-923b-3d50d53e9a1a.png)

Either way, I think just using `.wp-block-post-author` makes sense.  Updated.